src/LibraryManager.Contracts/FileHelpers.cs

```diff
@@ -245,21 +253,33 @@ public static bool DeleteFiles(IEnumerable<string> filePaths)
         /// Deletes folders if empty
         /// </summary>
         /// <param name="folderPaths"></param>
+        /// <param name="rootDirectory"></param>
         /// <returns></returns>
-        public static bool DeleteEmptyFoldersFromDisk(IEnumerable<string> folderPaths)
+        public static bool DeleteEmptyFoldersFromDisk(IEnumerable<string> folderPaths, string rootDirectory)
         {
+            if (folderPaths.Count() == 0)
+            {
+                return true;
+            }
+
+            HashSet<string> newFolderPaths = new HashSet<string>();
```

Comment:
Comparer for the hashset.

Reply:
added

---
In reply to: [200727652](https://github.com/aspnet/LibraryManager/pull/286#discussion_r200727652) [](ancestors = 200727652)